### PR TITLE
Allow multiple inline tests in a directory

### PR DESCRIPTION
**Pull request by rgrinberg**
The fix is to choose a unique name for the runner based on the library name

### CODE REVIEW

File: test/blackbox-tests/test-cases/inline_tests/dune-file.t/run.t [hunk 1]
```diff
@@ -0,0 +1,59 @@
+  $ dune runtest dune-file | sed "s/(lang dune .*)/(lang dune <version>)/" | dune_cmd sanitize
```
Comment (jeremiedimino) [inline test/blackbox-tests/test-cases/inline_tests/dune-file.t/run.t:1]: How does this test work?

Comment (rgrinberg) [reply test/blackbox-tests/test-cases/inline_tests/dune-file.t/run.t:6]: I've added a description to this test if that's what you meant.

Comment (jeremiedimino) [reply test/blackbox-tests/test-cases/inline_tests/dune-file.t/run.t:6]: Thanks. Yeah, I was trying to understand what this was testing.

Comment (jeremiedimino) [reply test/blackbox-tests/test-cases/inline_tests/dune-file.t/run.t]: While we are at it, I made the test a little bit more focused.

**Review by ghost — APPROVED** [review]
Looks good!

**Pull request merged**